stringzilla/pybind11.cpp

```diff
@@ -512,6 +519,11 @@ PYBIND11_MODULE(stringzilla, m) {
     auto py_str = py::class_<py_str_t, std::shared_ptr<py_str_t>>(m, "Str");
     py_str.def(py::init([](std::string arg) { return std::make_shared<py_str_t>(std::move(arg)); }), py::arg("str"));
     define_slice_ops(py_str);
+    py_str.def("__getitem__", [](py_str_t &s, py::slice slice) {
+        py::ssize_t start, stop, step, slice_length;
+        slice.compute(s.size(), &start, &stop, &step, &slice_length);
+        return s.sub(start, stop);
```

Comment:
This seems wrong. You are just dropping the `step`. Either report it, or implement it.